src/main/java/com/wrapper/spotify/SpotifyApi.java

```diff
@@ -923,13 +999,14 @@ public void setRefreshToken(String refreshToken) {
   }
 
   /**
-   * Get information about the user's current playback state, including track, track progress, and active device.
+   * Get information about the user's current playback state, including track or episode, progress, and active device.
    *
    * @return A {@link GetInformationAboutUsersCurrentPlaybackRequest.Builder}.
    */
   public GetInformationAboutUsersCurrentPlaybackRequest.Builder getInformationAboutUsersCurrentPlayback() {
     return new GetInformationAboutUsersCurrentPlaybackRequest.Builder(accessToken)
-      .setDefaults(httpManager, scheme, host, port);
+      .setDefaults(httpManager, scheme, host, port)
+      .additionalTypes("episode");
```

Comment:
Yeah, in hindsight that doesn't make a lot of sense. I think it would be the best to let the user decide about accepting additional types, so I'll remove the defaults.